.circleci/config.yml

```diff
@@ -40,9 +45,28 @@ jobs:
       - run: npm install
       - run: npm run build
 
+  deploy:
+    docker:
+      - image: circleci/node:9.11.2
+    steps:
+      - checkout
+      - restore_cache:
+          keys:
+            - v1-dependencies-{{ checksum "package.json" }}
+            # fallback to using the latest cache if no exact match is found
+            - v1-dependencies-
+      - run: npm install
+      - run: npm run deploy
+
 workflows:
   version: 2
   test-and-build:
     jobs:
       - test
       - build
+      - deploy:
+          requires:
+            - test
```

Comment:
'test' could be replaced by 'build' (once build depends on test), right?

Reply:
The `deploy` task doesn't need to wait for the `build` task. 

We check the `build` task on every PR because we found we can have the `test`s pass but the `build` be broken. 

But the `deploy` task runs its own build for deployment so waiting for `build` would make it take longer for no gain.